core/os/os.go

```diff
@@ -26,6 +27,8 @@ func (t OSType) String() string {
 		return "Ubuntu"
 	case Windows:
 		return "Windows"
```

Comment:
Whilst you're here, can you remove windows.

Reply:
I don't think we can unfortunately